### PR TITLE
chore(readme): adding event information to the documentation

### PR DESCRIPTION
**Pull request by VictorLessa**

### CODE REVIEW

**Comment by VictorLessa** [comment]
Adding event information to the documentation

File: README.md [hunk 1]
```diff
@@ -146,6 +146,22 @@ class UserRegisterEmail {
...
+
+  onCompleted(job, result) {}
+  onActive(job) {}
+  ...
+}
+
+module.exports = UserRegisterEmail;
+
+```
+
```
Comment (HigoRibeiro) [inline README.md:164]: Nice!!
I think you could remove the old event description, your description is better!

File: .coveralls.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+repo_token: rBCfZd8OfK9RgUqX1lq3PzBgGBSpkRFC5
```
Comment (HigoRibeiro) [inline .coveralls.yml:1]: Where you got this token?

File: .gitignore [hunk 3]
```diff
@@ -20,8 +20,6 @@ lib-cov
 # Coverage directory used by tools like istanbul
 coverage
 
-# Coveralls configuration file
-.coveralls.yml
```
Comment (HigoRibeiro) [inline .gitignore:24]: I think this is necessary now, can you tell me why?

File: README.md [hunk 4]
```diff
@@ -146,6 +146,22 @@ class UserRegisterEmail {
...
+  ...
+
+  onCompleted(job, result) {}
+  onActive(job) {}
+  ...
+}
+
+module.exports = UserRegisterEmail;
+
+```
```
Comment (HigoRibeiro) [inline README.md:163]: Nice one!! :rocket: 

I think you could remove the old event description.

**Review by HigoRibeiro — CHANGES_REQUESTED** [review]
Thanks for your contrib! :heart:

**Comment by jpedroschmitz** [comment]
Hey @VictorLessa, could you adjust the requests that @HigoRibeiro has made on his review?

We have already fixed the CI issue, so this token is no longer necessary. I recommend you remove it 💜

File: .coveralls.yml (hunk 2 above)
Comment (VictorLessa) [reply .coveralls.yml:1]: I will remove as requested

Comment (VictorLessa) [reply .coveralls.yml]: I added this pos token I couldn't get github actions to run without it

**Comment by VictorLessa** [comment]
I finalized the change and committed, do I need to do anything else? First time contributing

File: .coveralls.yml [hunk 5]
```diff
@@ -0,0 +1 @@
+repo_token:
```
Comment (jpedroschmitz) [inline .coveralls.yml:1]: This file can be deleted

File: yarn.lock [hunk 6]
```diff
@@ -3540,7 +3540,7 @@ set-getter@^0.1.0:
   dependencies:
     to-object-path "^0.3.0"
 
-set-value@^1.0.0, set-value@^3.0.0, set-value@^3.0.2:
+set-value@^1.0.0, set-value@^3.0.0, set-value@^3.0.1:
```
Comment (jpedroschmitz) [inline yarn.lock:3543]: This change isn't necessary.

**Review by jpedroschmitz — CHANGES_REQUESTED** [review]
Hey @VictorLessa, thanks for your update.

You can delete the `.coveralls.yml` file and also remove your change on the `yarn.lock`.

On the Readme, @HigoRibeiro suggested you remove the old event description. Can you do that? 💜 

If you need any help, send me a message on [Twitter](https://twitter.com/jpedroschmitz).

**Comment by VictorLessa** [comment]
I made the changes and gave a pull request

**Comment by jpedroschmitz** [comment]
Hey @VictorLessa, thanks 💜 
We're ready to merge it, but I need you to fix the conflict on the Readme!

**Comment by VictorLessa** [comment]
I resolved conflicts, what's the next step?

**Comment by VictorLessa** [comment]
Any indication?

**Comment by jpedroschmitz** [comment]
> Any indication?

Hey, I've fixed the issue with coveralls. As soon as #68 lands on master I'm going to update this PR and merge it

**Comment by jpedroschmitz** [comment]
@VictorLessa, can you update your PR, I'm not able to sync it with master.

**Comment by VictorLessa** [comment]
I updated my repository

**Comment by jpedroschmitz** [comment]
Have you pushed the commit?

**Comment by VictorLessa** [comment]
Yes!

**Review by jpedroschmitz — APPROVED** [review]
Thanks for the PR! 💜

**Pull request merged**